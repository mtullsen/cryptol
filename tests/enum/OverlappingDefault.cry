enum Foo = A | B | C | D

f : Foo -> [8]
f x =
  case x of
    A -> 0
    _ -> 1
    C -> 2
    D -> 3
