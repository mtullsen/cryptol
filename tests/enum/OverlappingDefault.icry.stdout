Loading module Cryptol
Loading module Cryptol
Loading module Main

[error] at OverlappingDefault.cry:5:3--11:0:
  Overlapping choices for default case:
    Pattern at OverlappingDefault.cry:7:5--7:6
    Pattern at OverlappingDefault.cry:8:5--8:6
    Pattern at OverlappingDefault.cry:9:5--9:6
